tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -343,6 +343,8 @@ lldbg_create_compile_unit_mdnode(LL_DebugInfo *db, int lang_tag, char *filename,
       llmd_add_i32(mdb, 1); /* emissionMode: FullDebug */
     llmd_add_md(mdb, *imported_entity_list);
     llmd_add_string(mdb, "");
+    if (!XBIT(120, 0x40000000))
+      llmd_add_i32(mdb, 2); /* nameTableKind: None */
```

Comment:
What about the other values (0,1)?

Reply:
IIRC, these are just provision of other accelerated table like `GNU` but llvm prefers it own kind of tables by default.